ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/passthrough/PassThroughHandler.java

```diff
@@ -50,10 +53,11 @@ public void handle(final RoutingContext context, final JsonRpcRequest request) {
   @Override
   public void handle(final RoutingContext context) {
     final HttpServerRequest httpServerRequest = context.request();
+    final String uri = httpRequestArgs.equals("/") ? httpServerRequest.uri() : httpRequestArgs;
```

Comment:
Would it be required to escape httpRequestArgs here? i.e. `URLEncoder.encode`?

Reply:
Also, to avoid any potential NPE, we can create a constant for "/" and do `DEFAULT_URI.equals(httpRequestArgs) ? ... `

Reply:
Ok I can update that. Thanks for the review.
Makes sense to encode, will do that as well.